web/docs/guides/websockets.md

```diff
@@ -0,0 +1,129 @@
+---
+title: WebSockets
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# WebSocket support
+Wasp provides a fully integrated WebSocket experience by utilizing [Socket.IO](https://socket.io/) on the client and server. We handle making sure your URLs are correctly setup, CORS is enabled, and provide a useful `useSocket` abstraction for use in React components.
+
+## Enable in Wasp file
```

Comment:
Could we, before we dive into details, give a quick overview of needed steps?
We could say smth like:
1. Define your socket logic on the server.
2. Declare you are using sockets in wasp file + connect it to the server logic you defined.
3. Use them on the client, in React, via `useSocket`.
4. Optional: defines types for messages.

Maybe there shouldn't be (4), this is just an idea.

Reply:
Yep I can add a quick overview